cloudbuild-provision-cluster.yaml

```diff
@@ -133,11 +133,26 @@ steps:
     - 'CLOUDSDK_COMPUTE_ZONE=${_CLUSTER_LOCATION}'
     - 'CLOUDSDK_CONTAINER_CLUSTER=${_CLUSTER_NAME}'
 
+# Install Istio 1.5 on the cluster so we can use Istio Authorization Policies.
```

Comment:
Can you open a new issue and add a TODO to remove this step once the version of Istio bundled with Cloud Run for Anthos is upgrade to 1.5?

Reply:
Done.